scripts/publish.js

```diff
@@ -5,7 +5,7 @@ shell.set('-e');
 shell.set('+v');
 
 // This needs to change when we are ready to publish
-const DOCKER_HUB_REPOSITORY = 'salesforce/salesforcedx';
+const DOCKER_HUB_REPOSITORY = 'jonnyhork/salesforcedx';
```

Comment:
I will change this back to `salesforce/salesforcedx` once we have settled on all the feed back for this PR, I was using my own repo for testing.